src/form.js

```diff
@@ -764,11 +741,13 @@ define([
             //    this.fixBrokenReferences(mug);
             //}
             if (mug.options.isODKOnly) {
-                this.updateError({
+                // is this a good candidate for "info" message level?
+                mug.addMessage(null, {
+                    key: 'form-odk-only-warning',
+                    level: 'warning',
```

Comment:
I think it's probably a net gain as most editors would autocomplete the ERROR or WARNING, but probably won't autocomplete a string as easily. I would say keep it as it's already in there even though JS isn't perfect.

Could also make the update method check the level against the known good values and have it throw an error so that we learn about a misspelling sooner rather than later